bindings/node/Cargo.toml

```diff
@@ -13,8 +13,8 @@ publish = false
 crate-type = ["cdylib"]
 
 [dependencies]
-napi = "3.5.1"
-napi-derive = "3.3.2"
+napi = "3.5.2"
+napi-derive = "3.3.3"
 mk-pass = { path = "../../mk-pass", features = ["clap"] }
 
 [build-dependencies]
```

Comment:
`napi-build` crate hasn't been actually bumped per PR description. Could possibly be a bug in dependabot since this is a dev-only dependency... I'll have to bump this manually.